setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 from distutils.core import setup
 
-version = "0.3.0"
+version = "0.3.2"
```

Comment:
I had missed a version bump on the last PR. So incremented twice this time around. :D